sdks/python/apache_beam/__init__.py

```diff
@@ -75,12 +75,6 @@
         'This version of Apache Beam has not been sufficiently tested on '
         'Python %s.%s. You may encounter bugs or missing features.' %
         (sys.version_info.major, sys.version_info.minor))
-  elif sys.version_info.minor == 8:
```

Comment:
We should also change the boundaries of the above condition to include 3.8

Reply:
+1

Reply:
fixed